pkg/uploadproxy/uploadproxy.go

```diff
@@ -252,8 +252,8 @@ func (app *uploadProxyApp) resolveUploadPath(pvc *v1.PersistentVolumeClaim, pvcN
 			path = common.UploadArchivePath
 		}
 	default:
-		// Escaping user-controlled strings to avoid cross-site scripting (XSS) attacks
-		return "", fmt.Errorf("rejecting upload request for PVC %s - upload content-type %s is invalid", html.EscapeString(pvcName), html.EscapeString(contentType))
+		// Caller is escaping user-controlled strings to avoid cross-site scripting (XSS) attacks
```

Comment:
Nit: Since we are keeping this comment it may make more sense to move it where we are escaping the strings.

Reply:
I changed it a bit with the intention to keep it here, but I don't mind. Did you catch the wording change?

Reply:
Waiting for the tests, I'll lgtm once they pass

Reply:
Yeah this time I ran them locally :stuck_out_tongue: